lib/es/esClient.js

```diff
@@ -1,11 +1,37 @@
 const elasticsearch = require('elasticsearch');
+const waitUntil = require('wait-until');
 
 const toBulkInsertArray = require('./toBulkInsertArray');
 const esConfig = require('../../config/esConfig');
 const log = require('../utils/logger');
 
+const pollingIntervalMs = Number(process.env.POLLING_INTERVAL) || 5000;
+const maxTimes = Number(process.env.MAX_TIMES) || 36;
+
 const client = new elasticsearch.Client(esConfig.getConnectionParams());
 
+function clusterIsHealthy(health) {
+  const status = health[0] && health[0].status;
+  // Note: yellow is the highest state for a single cluster
+  return status === 'yellow' || status === 'green';
+}
+
+function esServerReady() {
+  return client.cat.health({ format: 'json' }).then(clusterIsHealthy).catch(() => false);
+}
+
+function waitForEsToStart() {
```

Comment:
Can this be renamed to something like `waitForEsToBeReady` or `waitForHealthyEs`? It might be waiting for it to start in development but really it is waiting for the cluster state to be healthy, regardless of the environment in which it is running.